src/views/ControlPanel.vue

```diff
@@ -308,6 +308,21 @@ export default {
 					color: 'warning',
 					desc: 'Allows to shutdown the Zwave API to safely unplug the Zwave stick.',
 				},
+				{
+					text: 'Learn mode',
+					options: [
+						{
+							name: 'Start',
+							action: 'startLearnMode',
+							args: {
+								confirm:
+									'Initiate learn mode on primary controller first and then click OK here.',
+							},
+						},
```

Comment:
I would also add the stop action here

Reply:
Added
![image](https://github.com/user-attachments/assets/9d952974-a992-49a2-8877-5ce2ae322b4a)